app/views/promotion/weeklyLandingPage.scala.html

```diff
@@ -17,14 +17,15 @@
 @import com.gu.memsub.images.ResponsiveImageGenerator
 @import com.gu.memsub.BillingPeriod.SixWeeks
 
+@import com.gu.memsub.Product.WeeklyZoneA
 @(country: Country, catalog: Catalog, promoCode: Option[PromoCode], promotion: Option[PromoWithWeeklyLandingPage],description:Html, md: MarkdownRenderer)
     @anyPromotion = @{promotion.map(asAnyPromotion)}
     @isSixForSix = @{anyPromotion.filter(p => (catalog.weekly.plans.flatten.filter(_.charges.billingPeriod == SixWeeks).map(_.id).toSet intersect p.appliesTo.productRatePlanIds).nonEmpty)}
     @nonTrackingPromotion = @{anyPromotion.flatMap(p => p.asDiscount orElse p.asFreeTrial orElse p.asIncentive) orElse isSixForSix}
     @title = @{"The Guardian Weekly Subscriptions"}
     @defaultImage = @{ResponsiveImageGroup(None,None,None,ResponsiveImageGenerator("021b11f82c8fb43da5ef308dc6d6c5bf2fecb9c8/0_0_2560_300",Seq(2560)))}
     @discountedRegions = @{WeeklyPromotion.validRegionsForPromotion(promotion, promoCode, country)(catalog)}
-    @main(title = s"$title | The Guardian", bodyClasses = List("is-wide")) {
+    @main(title = s"$title | The Guardian", bodyClasses = List("is-wide"), product = Some(WeeklyZoneA)) {
```

Comment:
I had to pick a concrete implementation of weekly but actually I just want to say weekly in general